CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# v1.14.0-beta.1
```

Comment:
minor comment: I thought we decided to keep beta version changes and add a new section for the final release. That's at least what we did for 1.13.0

Reply:
Hmm, could go either way on this. I like clean release notes, but maybe we keep the changelog as is, but create final release notes for the github release that combines them?

Reply:
Ah - never mind, I see what you mean. Keep the beta sections but add a final section that incorporates.